### PR TITLE
Move from travis to github actions

### PR DESCRIPTION
**Pull request by jairhenrique**
- Move from travis to github actions
- Add python 3.9 and 3.10 support

### CODE REVIEW

**Comment by jairhenrique** [comment]
@kevin1024 @IvanMalison many tests are failing, I believe github actions will bring more visibility.

**Comment by kevin1024** [comment]
Oh cool.  All I have to do is merge this pr and the tests run in GitHub?

**Comment by jairhenrique** [comment]
> Oh cool. All I have to do is merge this pr and the tests run in GitHub?

@kevin1024 yes! You can see it in action here https://github.com/jairhenrique/vcrpy/pull/1

**Comment by kevin1024** [comment]
Rad.  Thanks!

**Comment by kevin1024** [comment]
Oh hey can you update the Travis button in the readme?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install project dependencies
+        run: |
+          python -m pip install -U pip codecov tox tox-gh-actions
```
Comment (cclauss) [inline .github/workflows/main.yml:34]: Is `-U` for `--user` or for `--upgrade`?  Both are valid pip options.  Using `--upgrage` makes intentions clear to the reader.

Comment (jairhenrique) [reply .github/workflows/main.yml:34]: @cclauss `-U` is for upgrade! `--user` has no alias.

``` 
-U, --upgrade               Upgrade all specified packages to the newest available version. The handling of dependencies depends on the upgrade-strategy used.
```

Comment (cclauss) [reply .github/workflows/main.yml:34]: I know that but does the ___reader___ know that?  Why should they be forced to guess?  My point is that `--upgrade` is more self-documenting than `-U`.

Comment (jairhenrique) [reply .github/workflows/main.yml:34]: @cclauss I get your point. Can you open a PR to improve this?

**Pull request merged**